src/main/java/gg/skytils/skytilsmod/mixins/transformers/util/MixinMouseHelper.java

```diff
@@ -18,16 +18,21 @@
 
 package gg.skytils.skytilsmod.mixins.transformers.util;
 
-import com.llamalad7.mixinextras.injector.WrapWithCondition;
-import gg.skytils.skytilsmod.mixins.hooks.util.MouseHelperHookKt;
+import gg.skytils.skytilsmod.mixins.hooks.util.MouseHelperHook;
 import net.minecraft.util.MouseHelper;
+import org.lwjgl.input.Mouse;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(MouseHelper.class)
 public class MixinMouseHelper {
-    @WrapWithCondition(method = "ungrabMouseCursor", at = @At(value = "INVOKE", target = "Lorg/lwjgl/input/Mouse;setCursorPosition(II)V", remap = false))
```

Comment:
I'm pretty sure wrap with condition is better because if other modifications are made it still works

Reply:
> I'm pretty sure wrap with condition is better because if other modifications are made it still works

no clue how wrap with condition works